programs/governance-rewards/src/instructions/escrow/create.rs

```diff
@@ -0,0 +1,51 @@
+use anchor_lang::prelude::*;
+use anchor_spl::token::{Token, TokenAccount};
+
+#[derive(Accounts)]
+pub struct CreateEscrow<'info> {
+    #[account(
+        init,
+        payer = payer,
+        seeds = [
+            realm.key().as_ref(),
+            admin.key().as_ref(),
+            b"escrow".as_ref(),
+            user.key().as_ref(),
+            mint.key().as_ref(),
+        ],
+        bump,
+        token::mint = mint,
+        token::authority = escrow_authority
+    )]
+    escrow: Account<'info, TokenAccount>,
+
+    /// CHECK: Not read
+    #[account(seeds = [b"escrow authority".as_ref(), realm.key().as_ref()], bump)]
+    pub escrow_authority: AccountInfo<'info>,
+
+    /// CHECK: Not read
+    realm: AccountInfo<'info>,
+
+    /// CHECK: Not read
+    mint: AccountInfo<'info>,
+
+    /// CHECK: Not read
+    user: AccountInfo<'info>,
+
+    /// CHECK: Not read
+    admin: AccountInfo<'info>,
```

Comment:
I was confused for a bit because I thought this `admin` and the one from the distribution were related. Could this be called differently? `escrow_authority`? (and rename what's currently called that to `escrow_vault_owner` or something?)

Reply:
I changed it to be more clear.